src/transformation/visitors/return.ts

```diff
@@ -102,7 +102,9 @@ export function createReturnStatement(
 function isInTryCatch(context: TransformationContext): boolean {
     // Check if context is in a try or catch
     let insideTryCatch = false;
-    for (const scope of walkScopesUp(context)) {
```

Comment:
I preferred this helper method, prevents a lot of error-prone mess, surely that was not that bad performance-wise